code/game/objects/items/devices/radio/headset.dm

```diff
@@ -213,6 +213,16 @@
 	icon_state = "headset_earmuffs"
 	item_state = "earmuffs"
 
+/obj/item/device/radio/headset/headset_earmuffs/syndie
+	name = "headset earmuffs"
```

Comment:
you don't need to paste any of these that are the same, they get inherited

Reply:
done, compiled, and tested